src/main/webapp/script.js

```diff
@@ -7,50 +7,39 @@ function loadDashboard() {
     $('#navbar').load('navbar.html', function() {
       $('#navbarDashboardSection').removeClass('d-none');
 
-      loadSortingDirection();
       getDashboardItems();
     });
 
     $('#itemSubmissionDiv').load('item-submission-dialog.html');
   });
 }
 
-/**
- * Loads the stored values for the sorting direction selector and adds a
- * callback to load the entertainment items when the selector changes value.
- */
-function loadSortingDirection() {
-  const sortingSelector = $('#sortingDirection');
-  const sortDir = localStorage.getItem('sortDir');
-
-  if (sortDir !== null) {
-    sortingSelector.val(sortDir);
-  }
-
-  sortingSelector.change(function() {
-    localStorage.setItem('sortDir', $(this).val());
-    getDashboardItems();
-  });
-}
-
 /**
  * Fetches entertainment items from DashboardServlet
  * to populate the Dashboard.
+ *
+ * @param { boolean } shouldRefresh - clears and loads the entertainment items
```

Comment:
maybe shouldClear or clearCurrentItems to indicate you're actually clearing

Reply:
I renamed the variable to clearCurrentItems to show the intent better.